.github/workflows/RequestReview.yml

```diff
@@ -15,13 +15,13 @@ jobs:
         github.event.pull_request.head.repo.full_name == github.repository
     steps:
       - id: secrets
-        uses: SonarSource/vault-action-wrapper@v3
+        uses: SonarSource/vault-action-wrapper@3996073b47b49ac5c58c750d27ab4edf469401c8 # 3.0.1
         with:
           secrets: |
             development/github/token/{REPO_OWNER_NAME_DASH}-jira token | GITHUB_TOKEN;
             development/kv/data/jira user | JIRA_USER;
             development/kv/data/jira token | JIRA_TOKEN;
-      - uses: sonarsource/gh-action-lt-backlog/RequestReview@v2
+      - uses: sonarsource/gh-action-lt-backlog/RequestReview@c686e352b369c4237e859acf335bf205a3926ebf # v2
```

Comment:
Same question as on sonar-cpp PR: Does this security recommendation apply to SonarSource actions as well? Is the cost of maintaining hashes whenever a SonarSource action is updated needed?

Reply:
for the records: https://github.com/SonarSource/sonar-cpp/pull/4706#discussion_r2071330505